Applications/InciSe/CMakeLists.txt

```diff
@@ -8,44 +8,26 @@ endif()
 # Create a cache entry for the provisioning file which is used to export
 # the file name in the MITKConfig.cmake file. This will keep external projects
 # which rely on this file happy.
-set(MITK_InciSe_PROVISIONING_FILE "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/MitkInciSe.provisioning" CACHE INTERNAL "InciSe provisioning file" FORCE)
+set(MITK_InciSe_PROVISIONING_FILE "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/InciSe.provisioning" CACHE INTERNAL "InciSe provisioning file" FORCE)
 
 set(_plugins
  org.mitk.gui.qt.stdmultiwidgeteditor
  org.mitk.gui.qt.datamanager
  org_mitk_planarfigure
```

Comment:
we may not need planar figure.

Reply:
Do you know exactly what this plugin does? If yes and you think it's useless I can remove it. Otherwise open an issue about it for us to remember to investigate it